expenser-server/middleware/auth.go

```
@@ -0,0 +1,64 @@
+		Error: err.Error(),
+	})
+	if err != nil {
+		log.Printf("Error marshalling JSON: %s", err)
+		w.WriteHeader(500)
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.Header().Set("Content-Length", fmt.Sprintf("%d", len(dat)))
+	w.WriteHeader(http.StatusUnauthorized)
+	w.Write(dat)
```

golangci-lint (1.61.0)

Error return value of `w.Write` is not checked

(errcheck)